src/applications/vaos/tests/e2e/covid19-vaccine.cypress.spec.js

```diff
@@ -182,3 +189,124 @@ describe('VAOS COVID-19 vaccine appointment flow', () => {
     cy.findByText(/Continue/i).should('not.exist');
   });
 });
+
+describe('VAOS COVID-19 vaccine appointment flow using VAOS service', () => {
+  it('should submit form', () => {
+    vaosSetup();
+
+    mockFeatureToggles({
+      v2Requests: true,
+      v2Facilities: true,
+      v2DirectSchedule: true,
+    });
+    mockLoginApi();
+    mockAppointmentsApi({ apiVersion: 0 });
```

Comment:
Other than removing the line above, LGTM!

Reply:
Version numbers are different.